lib/services/FirestoreUtils.dart

```diff
@@ -12,16 +12,17 @@ import 'package:myportion_app/model/Schedule.dart';
 class FireStoreUtils {
   static FirebaseFirestore firestore = FirebaseFirestore.instance;
   Reference storage = FirebaseStorage.instance.ref();
-  String userID;
-  String feederID;
-  String petID;
+  static String userID;
+  static String feederID = "temp";
+  static String petID = "temp";
```

Comment:
What's the scoop on this? Why did we have to initialize these rather than following the form of the `userID`?

Reply:
Should we initialize it to `""` if we have to initialize it here?

Reply:
This was a temporary change to prevent breaking, but I believe it can be done without in its current state. I will tests this and update accordingly.